CronofyCSharpSampleApp/Persistence/DatabaseHandler.cs

```diff
@@ -47,47 +48,24 @@ public static IEnumerable<T> Many<T>(string sql) where T : ITableRowModel, new()
             if (!_initialized)
                 Initialize();
```

Comment:
The logic for `Initialize` could be moved into a static constructor and then you wouldn't need to test this